beacon-chain/rpc/eth/validator/handlers_test.go

```diff
@@ -2668,6 +2747,7 @@ var (
       "aggregator_index": "1",
       "aggregate": {
         "aggregation_bits": "0x01",
+        "committee_bits": "0x01",
```

Comment:
Don't edit the pre-Electra object because that may make it invalid, make a new one instead